ceph_deploy/hosts/rhel/uninstall.py

```diff
@@ -7,7 +7,7 @@ def uninstall(conn, purge=False):
         'ceph-common',
         'ceph-mon',
         'ceph-osd',
-        'radosgw'
+        'ceph-radosgw'
```

Comment:
that's what we've been doing for a while.  yum just throws a warnings, but marches on.  I *do* think that a later enhancement we can make, post package-split everywhere, is uninstalling specific packages as well.  Though we should check to see if there is a real usecase for that.